### PR TITLE
Adding PlotAPI recipe

### PR DESCRIPTION
**Pull request by shahinrostami**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by shahinrostami** [comment]
Confirming that I'm willing to be listed here [x].

**Comment by shahinrostami** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Comment by shahinrostami** [comment]
@conda-forge-admin, please ping @conda-forge/staged-recipes

This is ready for review/merge

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Dr. Shahin Rostami
```
Comment (ocefpaf) [inline LICENSE.txt:3]: Why did you add this license here?

File: recipes/plotapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set name = "plotapi" %}
+{% set version = "6.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (ocefpaf) [inline recipes/plotapi/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/plotapi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/plotapi/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/plotapi/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: 352761ecff3c7cd36c15180af01df50f991c2a223e19b89dbb72306575dde7a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (ocefpaf) [inline recipes/plotapi/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: LICENSE.txt (hunk 1 above)
Comment (shahinrostami) [reply LICENSE.txt:3]: Thank you for catching this @ocefpaf. I used the GitHub front-end to generate the license and it places it in the repo root, regardless of the current dir. I'll delete it now.

File: recipes/plotapi/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.7.1,<4.0.0
```
Comment (ocefpaf) [inline recipes/plotapi/meta.yaml:20]: Upstream does not declare a min and max Python. Conda-forge's min is 3.6.
```suggestion
    - python >=3.6
```

File: recipes/plotapi/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python >=3.7.1,<4.0.0
```
Comment (ocefpaf) [inline recipes/plotapi/meta.yaml:22]:
```suggestion
    - python >=3.6
```

File: recipes/plotapi/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    - requests >=2.20.0,<3.0.0
```
Comment (ocefpaf) [inline recipes/plotapi/meta.yaml:23]: Upstream does not declared any dependencies but the code does use requests.

**Review by ocefpaf — CHANGES_REQUESTED** [review]

File: recipes/plotapi/meta.yaml (hunk 7 above)
Comment (shahinrostami) [reply recipes/plotapi/meta.yaml:23]: Thank you @ocefpaf . I've updated the version to the latest which includes `requests` as a declared dependency.

**Comment by shahinrostami** [comment]
@conda-forge-admin, please ping @conda-forge/staged-recipes

Thank you @ocefpaf for the review.

This is ready for re-review.

**Comment by shahinrostami** [comment]
@conda-forge-admin, please ping @conda-forge/core
CC @ocefpa 

The feedstock for PlotAPI seems to be empty https://github.com/conda-forge/plotapi-feedstock, any ideas?

**Comment by shahinrostami** [comment]
@conda-forge-admin, please rerender

**Comment by shahinrostami** [comment]
@conda-forge-admin, please rerender

**Comment by shahinrostami** [comment]
@conda-forge-admin, please restart ci

**Comment by ocefpaf** [comment]
> The feedstock for PlotAPI seems to be empty https://github.com/conda-forge/plotapi-feedstock, any ideas?

If you look at the staged-recipes main logs you'll see:

```
2022-09-06T18:11:03.3186650Z GitHub API Rate Limit Info:
2022-09-06T18:11:03.3187342Z ---------------------------
2022-09-06T18:11:03.3187664Z token:  GH_TRAVIS_TOKEN
2022-09-06T18:11:03.3188330Z Currently remaining 5000 out of 5000.
2022-09-06T18:11:03.3188660Z Will reset in 0:59:59.681555.
```

That means we are rate limited and feedstock creations will be delayed. There is nothing we can do but wait for the API rate limitation to reset.

**Comment by shahinrostami** [comment]
@ocefpaf ah, hopefully it has reset! Will it happen automatically, or does it need you to manually trigger? This is all I can see in the actions:

Screenshot 2022-09-06 at 19 38 34

**Comment by ocefpaf** [comment]
> Will it happen automatically,

It does. Like I said above. All we have to do is to wait.

**Comment by shahinrostami** [comment]
> > Will it happen automatically,
> 
> It does. Like I said above. All we have to do is to wait.

Hey @ocefpaf, I'm trying to make a little sense of this.  The section of the logs you quoted earlier:

> `2022-09-06T18:11:03.3188330Z Currently remaining 5000 out of 5000.`

Implies that it's _not_ being rate-limited, i.e. there are 5000 remaining out of a 5000 limit. 

I've been watching the GH Actions on `staged-recipes` for a few hours now and it appears that there's been no issue creating other feedstock repos, meanwhile the plotapi recipe has been stuck.

What I'm seeing now is this:
```
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.travis-ci.com/repo/24102281/setting/builds_only_with_travis_yml
```

All I can find on it https://travis-ci.community/t/loading-repositories-hangs-403-with-installation-missing-on-repos/6613/2
